### PR TITLE
updates to the ui for thursday.

### PR DESCRIPTION
**Pull request by jonhartzler**

### CODE REVIEW

File: src/components/layouts/HomePage/index.js [hunk 1]
```diff
@@ -46,7 +47,18 @@ export const Layout = ({
           </main>
         </BaseStyles>
         <FooterWrapper location={location}>
-          <div>Footer!</div>
+          <Link to='https://rarebooks.library.nd.edu'>
```
Comment (danthewolfe) [inline src/components/layouts/HomePage/index.js:50]: `Link` from Gatsby should only be used for internal Links. https://www.gatsbyjs.com/docs/reference/built-in-components/gatsby-link/

Time to discuss if we want to get rid of the custom version of `Link` from Internal/Link?

File: src/templates/home.js [hunk 2]
```diff
@@ -37,37 +81,33 @@ const Home = ({ data, location }) => {
         <section>
           <MultiColumn columns='2'>
             <Column>
-              <div sx={sx.topContent}>
-                <div dangerouslySetInnerHTML={{ __html: markdownRemark.frontmatter.contentTop }} />
+              <div sx={sx.topContentContainer}>
+                <p sx={sx.topContent} dangerouslySetInnerHTML={{ __html: markdownRemark.frontmatter.contentTop }} />
```
Comment (danthewolfe) [inline src/templates/home.js:85]: Depending on the content of the dangerous markdown, `<p/>` may not be technically valid. `<div/>` and `<p/>` should not be children of a `<p/>`.

**Review by danthewolfe — APPROVED** [review]
See two comments, but 👍

File: src/components/layouts/HomePage/index.js (hunk 1 above)
Comment (jonhartzler) [reply src/components/layouts/HomePage/index.js:50]: changed to an a  to match the nd link on the other side.

File: src/templates/home.js (hunk 2 above)
Comment (jonhartzler) [reply src/templates/home.js:85]: changed to a div.

**Pull request merged**